docs/proposals/34-observability2.0-opentelemtry-operator-and-collectors.md

```diff
@@ -0,0 +1,153 @@
+
+### Goals
+
+- **Deploy the OpenTelemetry Operator on Seed Clusters:** Install and configure the OpenTelemetry operator to manage otel-collector instances across shoot control plane namespaces.
+- **Deploy Gardener OpenTelemetry Collector Distribution in Shoot Control Planes:** Provision a dedicated Gardener OpenTelemetry collector distribution instance via the OpenTelemetry Operator in each shoot control plane namespace to handle observability data.
+- **Transport Logs signals through the OpenTelemetry Collector Distribution:** Configure fluent-bits to send logs to the OTel Collectors receivers and wire the otel collector exporters to the current logging backend.
+- **Replace valitail system service with otel-collectors on shoot nodes:** Replace current valitail system service on shoot nodes with otel-collector, keeping parity in the collected type of logs.
+
+### Non-Goals
+
+- **Immediate Replacement of Existing Tools:** This proposal does not aim to immediately decommission tools like Vali, Fluent-bit, or Prometheus; the migration will be phased.
+- **Unified Visualization Implementation:** Developing a single UI for logs, metrics, and traces is out of scope, as no mature open-source solution currently exists (per [*Observability 2.0*](https://github.com/gardener/logging/blob/master/docs/observability-2.0/Observability%202.0.md)).
+- **Full Tracing Adoption:** While tracing support is enabled, widespread instrumentation of Gardener components for distributed traces is a future step.
+
+---
+
+## Proposal
+
+This section outlines the implementation plan to introduce the OpenTelemetry operator and collectors into Gardener's shoot control plane namespaces and in shoot nodes as a systemd service, aligning with Step 2 of the [*Observability 2.0*](https://github.com/gardener/logging/blob/master/docs/observability-2.0/Observability%202.0.md) roadmap.
+
+### 1. Deploy the OpenTelemetry Operator on Seed Clusters
+
+- **Objective:** Establish centralized management of OpenTelemetry collectors across all shoot control planes.
+- **Details:** Deploy the OpenTelemetry operator (e.g., from the [OpenTelemetry Operator for Kubernetes](https://github.com/open-telemetry/opentelemetry-operator)) in the `garden` namespace of each seed cluster using `ManagedResources`, similar to the Prometheus operator deployment in [*GEP-19*](https://github.com/gardener/gardener/blob/master/docs/proposals/19-migrating-observability-stack-to-operators.md).
+- **Configuration:** The operator will be deployed following the `botanist` workflow in the same way as the current `prometheus` and `fluent-bit` operators. This means that during the `Seed` reconciliation flow the OpenTelemetryOperator will be deployed into the `garden` namespace.
```

Comment:
Which botanist workflow? Can you share a link?

Reply:
I am not aware of such link or docs.